tests/test_suite/unittests/test_find_modules.py

```diff
@@ -1,6 +1,6 @@
 from pathlib import Path
-
-from cosmic_ray.modules import find_modules
+from cosmic_ray.modules import find_modules, filter_paths
+from cosmic_ray.execution.local import excursion
```

Comment:
For tests, you should use the `path_utils` fixture to get `excursion`. Ultimate the excursion definitions in `path_utils` and `execution.local` should probably be combined.